bin/bundle-extensions

```diff
@@ -1,11 +1,10 @@
 #!/usr/bin/env python
 
 import os
-from subprocess import call
 from distutils.dir_util import copy_tree
 
-from pkg_resources import iter_entry_points, resource_filename, resource_isdir
-
+import importlib_metadata
+import importlib_resources
```

Comment:
I've moved to those two libraries since they are the backports of importlib packages that are in Python 3's stdlib while setuptools' pkg_resources is considered being deprecated.